code/game/world.dm

```diff
@@ -385,6 +389,22 @@ GLOBAL_VAR(restart_counter)
 /world/proc/on_tickrate_change()
 	SStimer?.reset_buckets()
 
+/world/proc/init_byond_tracy()
+	var/library
+
+	switch (system_type)
+		if (MS_WINDOWS)
+			library = "prof.dll"
+		if (UNIX)
+			library = "libprof.so"
+		else
+			CRASH("Unsupported platform: [system_type]")
```

Comment:
Realistically is this else branch needed?

Reply:
Preference thing. It's unused in practice (and it's unlikely Lummox supports Mac or anything else) but in principle stuff like system type should be treated as a non-exhaustive enum IMO

Reply:
you can't forget about the Plan 9 gamers, @AffectedArc07

Reply:
> you can't forget about the Plan 9 gamers, @AffectedArc07 

They don't deserve this luxury